### DISCUSSION TITLE
Unable to correctly define metadata and identify machines for multiple clients on a server

### DISCUSSION THREAD

**Discussion by Stupeflip** [jchristn/WatsonTcp#256] [Q&A]
![image](https://user-images.githubusercontent.com/40721820/236634825-05cd42e6-c3b0-4f0c-bf0d-6c66f438c3a7.png)

It is impossible to define the metadata correctly. The way you gather a client through the guid is not compatible with what I need.

On the one hand, I'm running a server and on the other hand, I need to let a lot of clients run, from different machines, and I need to identify the machines correctly and I just can't.

**Comment by jchristn** [comment]
What exactly is the issue with the library here?  The client is identified in the ```ConnectionEventArgs.Client``` by either ```Guid``` or ```IpPort```.
